code/game/objects/items/food/misc.dm

```diff
@@ -963,3 +963,22 @@
 	icon_state = "ready_donk_warm_mex"
 	tastes = list("enchiladas" = 2, "laziness" = 1)
 	foodtypes = GRAIN | DAIRY | MEAT | VEGETABLES | JUNKFOOD
+
+/obj/item/food/pierogi
+	name = "pierogi"
+	desc = "A Polish dumpling made by wrapping unleavened dough around a savoury or sweet filling and cooking in boiling water. This one is filled with a potato and onion mixture."
```

Comment:
Pierogies are popular across Eastern Europe. They are not merely a Polish food, but an Eastern European food.

Reply:
Ah yeah, true!